tools/system_libs.py

```diff
@@ -1363,7 +1363,7 @@ def handle_reverse_deps(input_files, only_forced):
   # If we are only doing forced stdlibs, then we don't know the actual
   # symbols we need, and must assume all of deps_info must be exported.
   # Note that this might cause warnings on exports that do not exist.
-  if only_forced:
+  if only_forced and not shared.Settings.BOOTSTRAPPING_STRUCT_INFO:
```

Comment:
this change looks separate? lgtm aside from this, which maybe I don't understand.

Reply:
Actually it is needed right now because of the current usage of `EMCC_ONLY_FORCED_STDLIBS`.   I'll be able to remove this once my we stop setting `EMCC_ONLY_FORCED_STDLIBS` here (i.e. once #13422 lands).

This one I think should land first though.

Reply:
The reason is that `EMCC_ONLY_FORCED_STDLIBS` otherwise forces all the contents of deps_info to be included, which results in a whole bunch of undefined symbols (which were previously being ignored).